packages/app/components/Schedule/SchedulePanel.tsx

```diff
@@ -43,6 +44,7 @@ export default function SchedulePanel({
           new Date(today.getFullYear(), today.getMonth(), today.getDate(), 8)
         }
       />
+      <UpdateCalendarButton courseId={courseId} />
```

Comment:
Should we conditionally render based on `role === Role.PROFESSOR` here instead of rendering `null` in the UpdateCalendarButton component?

Reply:
yeah I was debating between having the hook in Schedule or UPdate, but I think it's also more readable if I put it in Schedule